packages/task-utils/task-utils.js

```diff
@@ -0,0 +1,36 @@
+var matchedLastHistory = function(task, state) {
```

Comment:
I'm surprised we want to keep anything in `/packages/`.

Reply:
We don't. This shouldn't exist. Deleting.